File: Programming Basics/JavaScript/Conditional Statements/Exercises/toy-shop.js
function solve(input) {
    const tripPrice = Number(input[0]);
    const puzzles = Number(input[1]);
    const dolls = Number(input[2]);
    const bears = Number(input[3]);
    const minions = Number(input[4]);
    const trucks = Number(input[5]);

    let profit = puzzles * 2.6 + dolls * 3 + bears * 4.1 + minions * 8.2 + trucks * 2;
    if (puzzles + dolls + bears + minions + trucks >= 50) profit *= 0.75; // Make discount of 25%

    profit *= 0.9; // Pay rent (10% of the profit)

    if (profit >= tripPrice) console.log(`Yes! ${(profit - tripPrice).toFixed(2)} lv left.`);
    else console.log(`Not enough money! ${(tripPrice - profit).toFixed(2)} lv needed.`);
}

File: Programming Basics/C++/Conditional Statements/Exercises/ToyShop/ToyShop.cpp
#include <iostream>
using namespace std;

int main()
{
	// 1. Read input
	double tripPrice;
	int puzzles, dolls, bears, minions, trucks;
	cin >> tripPrice >> puzzles >> dolls >> bears >> minions >> trucks;

	// 2. Calculate profit
	double profit = puzzles * 2.6 + dolls * 3 + bears * 4.1 + minions * 8.2 + trucks * 2;
	if (puzzles + dolls + bears + minions + trucks >= 50)
		profit *= 0.75; // Make discount of 25%

	profit *= 0.9; // Pay rent (10% of the profit)

	// 3. Print output
	cout.setf(ios::fixed);
	cout.precision(2);
	
	if (profit >= tripPrice) cout << "Yes! " << profit - tripPrice << " lv left." << endl;
	else cout << "Not enough money! " << tripPrice - profit << " lv needed." << endl;
}
